idd/Energy+.idd.in

```diff
@@ -18494,11 +18761,11 @@ ZoneProperty:UserViewFactors:BySurfaceName,
        \memo (Number of Surfaces)**2 are expected. Any omitted surface pairs will be assumed to have a view factor of zero.
        \extensible:3 - copy last three fields, remembering to remove ;
        \format ViewFactor
-  A1,  \field Zone or ZoneList Name
-       \note View factors may be entered for a single zone or for a group of zones connected by Construction:AirBoundary
+  A1,  \field Space or SpaceList Name
+       \note View factors may be entered for a single space or for a group of spaces connected by Construction:AirBoundary
        \type object-list
-       \object-list ZoneNames
-       \object-list ZoneListNames
+       \object-list SpaceNames
+       \object-list SpaceListNames
```

Comment:
Ah, so here you switched the view factors input to spaces, which is by space name. So for backward compatibility the space name must match the zone name when not entered? If so then the note in the Zone object is wrong.